### PR TITLE
Wip use storage class

### PR DESCRIPTION
**Pull request by FaKod**
Added the possibility to use a given StorageClass

### CODE REVIEW

**Comment by FaKod** [comment]
What do you think about this PR?

**Comment by stevesloka** [comment]
What's the use-case to provide your own storage class? Right now it's used to scale PV's across zones since the operator creates them for you.

**Comment by FaKod** [comment]
The use case is: non AWS environments like bare metal or, as in my case, other cloud provider without something similar to EBS. The ability to provide a storage class is a great more on flexibility without disturbing other configurations.

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 .PHONY: all build container push clean test
 
 TAG ?= 0.0.6
-PREFIX ?= upmcenterprises
+PREFIX ?= innoq
```
Comment (stevesloka) [inline Makefile]: Could you remove this reference to your repo?

File: README.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ Following parameters are available to customize the elastic cluster:
 - [storage](https://kubernetes.io/docs/user-guide/persistent-volumes/)
   - type: Defines the type of storage to provision based upon cloud (e.g. `gp2`)
   - storage-class-provisioner: Defines which type of provisioner to use (e.g. `kubernetes.io/aws-ebs`)
+  - storage-class: Defines the name of an existing StorageClass object to use
```
Comment (stevesloka) [inline README.md]: Could you add a bit about how this is optional? Or something about it's use case, I don't want to confuse folks to think this is required, but I understand there are use-cases.

**Comment by stevesloka** [comment]
Hey, @FaKod I added a few nit comments. Would you mind cleaning those up, then rebase and squash your commits? Thanks!

**Comment by FaKod** [comment]
for simplicity created #85 
closing this one

**Pull request closed without merging**